### PR TITLE
PIM-8428 PIM displays `pim_common.code` on grids

### PR DESCRIPTION
**Pull request by pyklein**
PIM-8428 update changelog

PIM-8428 PIM displays  on grids

**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | OK
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: CHANGELOG-3.1.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 # 3.1.5 (2019-05-28)
 
+## Bug fixes
+
+- PIM-8426: User plainPassword validation doesn't work
```
Comment (jmleroux) [inline CHANGELOG-3.1.md]: Incorrect changelog.

File: src/Akeneo/Pim/Structure/Bundle/Resources/config/datagrid/attribute.yml [hunk 2]
```diff
@@ -67,7 +67,7 @@ datagrid:
             columns:
                 code:
                     type:      string
-                    label:     pim_common.code
+                    label:     Code
```
Comment (jmleroux) [inline src/Akeneo/Pim/Structure/Bundle/Resources/config/datagrid/attribute.yml]: Will it be translated ?

Comment (pyklein) [reply src/Akeneo/Pim/Structure/Bundle/Resources/config/datagrid/attribute.yml]: Yes! 
it goes into UIBundle/Resources to translate

File: CHANGELOG-3.1.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 # 3.1.5 (2019-05-28)
 
+## Bug fixes
+
+- PIM-8428 PIM displays pim_common.code on grids
```
Comment (Doodoune) [inline CHANGELOG-3.1.md]: need a rebase, as it's on 3.1.7

**Pull request merged**